### PR TITLE
converge towards plugin requirements

### PR DESCRIPTION
**Pull request by mfleader**
## Changes introduced with this PR

Converge towards plugin requirements.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by BATMAN-JD — APPROVED** [review]

File: .github/workflows/carpenter_build.yaml [hunk 1]
```diff
@@ -1,14 +1,12 @@
 name: Carpentry
 on:
-  push:
-    branches:
-      - main
   pull_request:
 env:
-  IMAGE_NAME: arcaflow-plugin-elasticsearch
-  IMAGE_TAG: 'latest'
+  IMAGE_NAME: arcaflow-plugin-uperf
```
Comment (portante) [inline .github/workflows/carpenter_build.yaml:5]: Why was this change merged?  This appears to be in the `elasticsearch` plugin repository, no?

File: .github/workflows/carpenter_push.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: Carpentry Push
+on:
+  push:
+    branches:
+      - main
+env:
+  IMAGE_NAME: arcaflow-plugin-uperf
```
Comment (portante) [inline .github/workflows/carpenter_push.yaml:7]: Shouldn't this have been `elasticsearch` instead?

File: .github/workflows/carpenter_build.yaml (hunk 1 above)
Comment (portante) [reply .github/workflows/carpenter_build.yaml:5]: Looks like this was fixed by #5.

File: .github/workflows/carpenter_push.yaml (hunk 2 above)
Comment (portante) [reply .github/workflows/carpenter_push.yaml:7]: Looks like this was fixed by #5.

**Pull request merged**